pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -72,13 +72,6 @@ python3.pkgs.buildPythonApplication rec {
         "docs/yaml/objects/dep.yaml"
```

Comment:
While touching Meson, we might also want to drop this patch since it is unmerged and adds a new feature to Meson, which might be confusing. The [issue this was added for](https://github.com/NixOS/nixpkgs/pull/171069#issuecomment-1209504841) was fixed in Bubblewrap: https://github.com/containers/bubblewrap/commit/d8b4221d540116cf927ecc44018cd6d2f2d0dd01